source/JABHandler.py

```diff
@@ -43,6 +43,9 @@
 import config
 from utils.security import isRunningOnSecureDesktop
 
+#: Verification of the architecture of the running system
+is_64Bit = os.environ["PROCESSOR_ARCHITECTURE"].endswith("64")

```

Comment:
As requested earlier, given https://github.com/nvaccess/nvda/issues/16330, could you make this a system wide constant in winAPI.constants?